src/app/static/src/app/components/figures/filteredData.ts

```diff
@@ -10,11 +10,7 @@ export interface FilteringProps {
 
 export function useFiltering(props: FilteringProps) {
 
-    const filterBySettings = (row: any) => {
-        if (!props.settings) {
-            return true;
-        }
-
+    const settings = computed<Dictionary<string | number> | null>(() => {
```

Comment:
Unrelated to the changes here, just realised the settings were being computed on every iteration of the filtering logic, when they could be their own computed variable.